docs/deployment/teams.md

```diff
@@ -315,21 +315,56 @@ To set a member as a manager for a subteam:
 
 ## Appendices
 
-### Member-scoped access tokens
+### Tokens
 
-Both members and admins can log in through the command-line interface (CLI) by entering the following command:
+An access token is a secure credential that authorizes a user to interact with the Semgrep AppSec Platform or API without requiring a username and password. Access tokens identify who is making a request and define what actions they are allowed to perform, based on the [token’s scope](#token-scopes).
 
-```
+Semgrep supports several types of access tokens:
+
+#### User-generated (Web API) tokens 
+
+These tokens are created by admins in Semgrep AppSec Platform. They are used for API access, integrations, and automation. 
+
+Some features of these tokens:
+- Not tied to a specific user account and remain valid until manually revoked, even if the creator is no longer associated with the deployment.  
+- Only admins can generate or manage these tokens. This can be done by going to **Settings → Tokens** in Semgrep AppSec Platform.  
+
+#### CLI tokens (Member-scoped) tokens
+
+These tokens authenticate users running scans or publishing rules from the CLI. A CLI token can be created by running the following command:
+
+```bash
 semgrep login
 ```
 
-This generates a unique token that is used to identify a member or admin. When logged in, members can run scans on their local machine through the `semgrep ci` command and publish a rule. This sends findings data to Semgrep AppSec Platform.
+Both members and admins can create CLI tokens. Once logged in, users can run scans on their local machine through the `semgrep ci` command and publish a rule. This sends findings data to Semgrep AppSec Platform.
+
+
+Some features of CLI tokens: 
+- Tied to the user who created them, but permissions cannot be elevated. For admin access, users must first obtain the admin role and then create a new token as an admin. See [Changing a user's role](#change-a-users-role).
+- Visible only to admins, not members, in **Settings → Tokens**.  
+- Work only for running scans and reporting results.  
+- Tokens generated with `semgrep login` are CLI-only and cannot be converted to Web API tokens.  
+- Running `semgrep logout` removes the local token but does not invalidate it server-side.  
+- CLI tokens record which user generated them, but actions are attributed to the token, not the individual user. 
+
+
+#### Agent tokens 
+
+Automatically generated when onboarding repositories for CI/CD scans. These tokens authenticate agents running automated scans within CI environments.  
+
+#### Temporary tokens
+
+Short-lived tokens generated internally for specific features, such as Managed Scans and Click-to-Fix. Most of these features are in beta.
+- Designated as temporary and typically deleted automatically through cleanup processes.  
+- Values are not exposed for reuse after creation.  
```

Comment:
I think these have actually recently been removed from display, so we may or may not want to mention them; if we do, we should mention whether or not they're visible. I'd recommend checking with the eng team internally.